packages/core/src/factory.ts

```diff
@@ -35,7 +35,7 @@ export function factory<Resource, Input, Output, Config>(
             Either.fold(
               error => {
                 // rethrow error we if we're attempting to mock
-                if ((configObj as IPrismConfig).mock) {
+                if (((configObj as unknown) as IPrismConfig).mock) {
```

Comment:
I know this is not ok, but the amount of templating in Prism is ridiculous and just drives me crazy. For now we will have to stick with these horrible things, as we proceed I will try to remove them all.